src/ui/public/notify/__tests__/notifier.js

```diff
@@ -340,6 +353,10 @@ describe('Notifier', function () {
       expect(notify('banner').title).to.equal('Attention');
     });
 
+    it('sets truncation length to 250', function () {
```

Comment:
Nit: change the description of the `it` to include "by default" at the end, just like the lifetime one right below it.